schemas/+acq/checkFileversion.m

```diff
@@ -0,0 +1,18 @@
+
+function oldversion=checkFileVersion(fn)
+
+oldversion = true ;
+try
```

Comment:
Huh... works fine for the file you mention on at-ssp, Matlab R2016b. Is h5readatt working for other files ?